region/crateria/central/Parlor and Alcatraz.json

```
@@ -924,6 +1091,72 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [3, 6],
+      "name": "Come in Shinecharged, Leave With Spark (Bottom Position)",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 20
+        }
+      },
+      "requires": [
+        "canShinechargeMovementComplex",
+        {"shinespark": {"frames": 22, "excessFrames": 0}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {
+          "position": "bottom"
+        }
+      },
+      "unlocksDoors": [
+        {
+          "types": ["super"],
+          "requires": []
+        },
+        {
+          "types": ["missiles", "powerbomb"],
+          "requires": [
+            "never"
+          ]
+        }
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Shoot while entering the room, to open the door.",
+        "Activate the spark immediately after landing, at a position low enough to pass underneath the platform."
+      ]
+    },
+    {
+      "link": [3, 6],
+      "name": "Come in Shinecharged, Leave With Spark (Top Position)",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 90
+        }
+      },
+      "requires": [
+        "canShinechargeMovementComplex",
```

```suggestion
        "canShinechargeMovementComplex",
        "canCarefulJump",
```
It's kinda like trickyJump movement, but not hard to do.